### ISSUE TITLE
How to get up in the morning when it's cold and you're exhausted

### ISSUE THREAD

**Issue by iteles** [dwyl/sleep#32]
Winter is here and heating has to be moderately low overnight or the air dries out and we all wake up with a headache.

I've been trying to make sure I switch off the lights by 22h30 every night (which I've so far done for 12 nights in a row) but am still battling the usual insomnia that keeps me awake past midnight. Result = I'm always exhausted when I wake up in the morning.

There are lots of things to be done for better sleep (as per this repo), but what can be done when you've already woken up and you're exhausted to help you cope? (Both practical tips and 'one must have a plan' type tips very welcome!)

**Comment by nelsonic** [comment]
@iteles there is quite a lot to "unpack" in this question, and you might not "like" the answer. 🙄 

## Winter? ❄️ 

Winter is an _inevitable_ fact of life for those of us who choose to live far from the [equator](https://weather.com/wunderground/news/news/winter-loses-meaning-at-the-equator).
During these months the days are shorter, often overcast and considerably colder 🌨 
it's like a trifecta of factors that make us want to curl up and hibernate like a bear in a cave. 🐻 💤 
Unless you are a migratory bird or a wealthy person who can "fly south" for the winter and enjoy back-to-back warm seasons, Winter is a reality you will need to come to terms with. 

### What to _Do_ About it?

Based on what I've read in _numerous_ well-researched/written books on sleep, exercise & nutrition,
there are a number of things you can do to maximise your chance of feeling _wide awake_ in the morning.

My best "cure" for the winter is my (_all-year-round_) **Morning Routine**:
+ High Intensity **Exercise** shortly after getting up to warm up and get blood flowing. 
(I do **5 minutes** of exercise _first_ thing. I find this is "enough" to _feel_ warmed up. [sample routine](https://github.com/nelsonic/nelsonic.github.io/blob/master/draft/goals.md#132-sample-morning-exercise-routine-revisited))
+ Drink Tea or **Warm Water** immediately after exercising. Warming from both the inside and hands is a great way to _feel_ warm and also hydration is _super_ important for waking up and feeling fresh. 
I drink a **litre** of **warm lemon water _first_ thing** it has _numerous_ benefits for kickstarting my day.
+ **Warm clothes** e.g: "Fleece" are good for _maintaining_ body heat. If you don't have a _super_ warm fleece, get one. The @dwyl hoodies are "OK", but I usually find myself wearing my [Patagonia fleece](https://www.patagonia.com/shop/womens-fleece) (_which I bought second-hand on Ebay for £18 but still has plenty of "life" left in it!_)
+ **Full-spectrum reading/work light** (_I'm basically obsessed with having a **mega-bright** "mini sun" light wherever I am in the morning._) 
+ Get as _much_ **natural** (_direct sunlight_) **light** during the day as you can. 
  + If it's not raining, make a _habit_ to take a "break" from sitting at a desk indoors and get _outside_ for a walk or read in the _Sun_. Even if it's overcast you still get 20-30% of the sun's beneficial [UV-B](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3427189) rays which will "recharge" your [mitochondria](https://en.wikipedia.org/wiki/Mitochondrion) and help you fall asleep easier at night because your body will naturally produce more [Melatonin](https://en.wikipedia.org/wiki/Melatonin) if your eyes and skin are exposed.
  I usually take _off_ my fleece/jumper (_even if I feel cold_) to get more UV on my skin. 
Feeling "cold" for a few minutes is _good_ for circulation see: https://en.wikipedia.org/wiki/Cryotherapy
When you put your jumper back on and/or head back indoors you will find that your _core_ temperature will increase as your body _compensates_ for feeling cold. If you are unable to _regulate_ your body temperature, do 10-20 squats and you'll find yourself warming up _fast_!
+ Have a **pre-defined _clear_ task** for first thing in the day. Not "checking email" or any other work that someone **`else`** wants/needs you to do, your _own_ work<sup>1</sup>. 

## "Lights Out" at 22:30 🕥 

This is an _excellent_ start and it's worth maintaining this ["streak"](https://github.com/nelsonic/nelsonic.github.io/issues/621) and making it a _consistent_ routine.
Winter is the _perfect_ time to get to bed even _earlier_ than 22:30 because our natural circadian clock lends itself to going to _sleep_ a few of hours after sunset which is _much_ earlier in Winter.
I've found that my optimal go to bed time is **21:30** in the winter.
And on days where I have completed my work, done "_enough_" exercise and had moderate sun exposure I fall asleep quite promptly. (_this was definitely not the case when living in London where the road noise, inescapable street lighting and "stress" was silly and I never got to bed before midnight...!_)

Keep the good bedtime up, it will help you wake up in the morning.
But if you don't **_fall_ asleep** reasonably promptly after getting to bed by 22:30
**`.then`** you may need to retrace the steps you made during the day.<sup>2</sup>

## The _Many_ Benefits of Waking Up "_Early_". ☀️🕔 

A few of the _biggest_ advantages of waking up and _getting_ up "_early_" in the morning include:
+ An ***instant*** sense of "***accomplishment***" that you are "***ahead***" on the day. 
  Ahead of your always-tired (_unfocussed_) peers who stayed up late watching TV or "socialising".
  Ahead of the _mediocre_ majority of people who hit the "***snooze button***" and only get up 5 minutes _after_ they needed to leave the house to be at their "job" on time. 
+ Get **2 hours** of "***deep***" (_focus_) work done **`before`** the "chatter" of the day starts.
Most people don't get **2 hours** of "_deep_" work done their _whole_ day. Focused "_deep_" work is _rare_. _Most_ people only do [_shallow_](https://www.amazon.com/Shallows-What-Internet-Doing-Brains/dp/0393339750) work and never achieve anything _meaningful_, that's a _shame. 
  + I learned a long time ago (_when I was 17, just-in-time to cram for exams_) that an hour of "quite time" in the early morning is worth 5 hours of day-time where noise and interruptions are rife.
+ ***Momentum***. Once you have _started_ with focus work _first_ thing in the morning, it's _much_ easier to ***`continue`*** until "lunch". If you can do 06:00 - 12:00 of deep work each day you will accumulate (_assuming 6 days a week_) 36 hours/week. If you got 36h of deep work each week you would rapidly "catch up" and even _surpass_ some of the most prolific creative people!
This leaves the _afternoon_ for any "admin" or "shallow" work that still _needs_ to get done but isn't anywhere _near_ as important to the _long-term_. 

# tl;dr - _Practical Tips_

## `#1`: Make it a _Game_!

> "_In every job that must be done, there is an element of **fun**. 
You **find the fun** and - SNAP - the **job's a game**!_" ~ [Mary Poppins](https://youtu.be/HrnoR9cBP3o)

Figure out how to [_gamify_](https://en.wikipedia.org/wiki/Gamification) your life.
If you treat your life as a game, with clear rules that you can "_win_". All you have to do is _play_ & _have fun_!
If you ever feel that something is "unfair", remember that there are at _least_ a billion people for whom the "game" is _considerably less "fair_.<sup>3</sup>

> **Note**: I'm _not_ saying that you should play you life like a "chess game" where other humans are your "pawns" ... rather think of our life as an "Open World Collaborative Platformer" where you want _everyone_ on your team to "level up" _with_ you so that you can keep _wining_ at life!

## `#2`: Track Your Wake Time

(_ideally_) Use a paper "sleep" notebook that never leaves your bedside to note the time you woke up in the morning. Simply writing the date and time is "enough" to start with. You can reflect on the data in a no/low-tech way. If you _prefer_ to use an "app" for this, want to _pair_ on building it?

I use the ["screenshot"](https://support.apple.com/en-us/HT204673) feature of my Apple Watch to "grab" my wake-up time as soon I wake up. e.g:
![image](https://user-images.githubusercontent.com/194400/51806156-44f27c80-226e-11e9-80a2-d3dcd0ffe647.png)
"close enough" to 05:00, thanks body clock. 🕔 ✅ 
Logging your "wake time" is your _first_ accomplishment for the day.

## `#3`: Get Up!

Get up when you _wake_ up. Don't lay in bed staring at phone or "thinking" about getting up ...
If you have woken up naturally, your circadian clock has brought you to consciousness at an appropriate time. Get up now! _Especially_ if you "don't feel like it".  You will _feel_ accomplished.
Do what you can for a _long_ as you can. If you need a _nap_ (_e.g: after lunch_) go for it! 
Just make sure you _log_ it in your "sleep journal" (_see above_).

## `#4`: _Never_ "Snooze". 

Learn to live _without_ an alarm clock. (_as you know I haven't used an alarm clock for **years** and I still get up around 05:00 like "clockwork"_). But if you feel that you "_need_" an alarm clock ***never*** under _any_ circumstances hit the "***snooze***" button. It sends _totally_ the "wrong message" your body. 

If you still feel "tired", remind yourself of your _mission_ and how it is more _important_ than your tiredness, cold or any other short-term "discomfort". Until you are able to put a mission above your own immediate comfort, you aren't "ready" to make the most of your life. 

# _Remember_

The _first_ 3 days of an "early riser" life/streak are by _far_ the most difficult. It get's _much_ easier!
Once you get into a _routine_ waking/getting up "early" and being a "morning person" will come _naturally_.


<hr />

><sup>1</sup> If you feel that you don't  have a ***clear objective*** to work towards, speak to me, I can help!
> I know _exactly_ what I want to achieve in _life_ and what I need to learn/do along the way.
> _Most_ people don't know what they want in life, so they let advertisers spoon-feed them their wants and "needs". But the _reason_ people don't know what they want is not because it's "_difficult_" to distill life objectives, rather aimless people **_avoid_ thinking** about it at **_all_ cost**; _constantly_ doing everything they can to ***not*** think about the one thing that is ***inevitably*** in life: **death**. 💀 

><sup>2</sup>Retracing the day is a good way to end the day and ensure that you have ***captured*** everything from your mind into a reliable system rather than letting it "_play_" on your mind and keep you awake _long_ after the 22:30 "light's out". 

><sup>3</sup>According to the most recent data, **14.7%** of the **world population** remains ***illiterate***.
That means that you are "better off" than a (14.7% x 7.7 billion) 1.13 Billion People who are _systematically_ disadvantaged in life because they [cannot _read_](https://github.com/nelsonic/nelsonic.github.io/issues/521#issuecomment-451088965)! see: [ourworldindata.org/literacy](https://ourworldindata.org/literacy)

**Comment by iteles** [comment]
Thanks for putting so much into this answer @nelsonic (although of course I'm not surprised, you're the 👑 of the thorough answer).

The only way I reckon this will work for me is by incorporating a few of those things: having a first task to do for the day (which I already always have) but most importantly just getting up when the alarm clock goes off and not snoozing - for now this will have to be sheer willpower (in short supply for me early in the morning), thought gamifying it (with a non-food 'reward') might help! 👍 🙌
